build.js

```diff
@@ -32,7 +31,27 @@ var path = require('path');
 var cheerio = require('cheerio');
 // var child_process = require('child_process');
 
-var useCompilers = String(process.argv[2]).toLowerCase() === "compilers";
+var useCompilers = false;
+var excludeCurrentBrowser = false;
+var environments;
+
+process.argv.slice(2).forEach(function(arg) {
+	var parts = String(arg).toLowerCase().split('=');
+
+	switch (parts[0]) {
+		case 'compilers':
+			useCompilers = true;
+			break;
+		case 'environments':
+			environments = require(parts[1]);
+			break;
+		case 'excludecurrent':
+			excludeCurrentBrowser = true;
+			break;
+	}
+});
+
+if (!environments) environments = require('./environments');
```

Comment:
this seems like it could be handled as a default in the args parser

Reply:
I assume you mean a default value when destructuring  the returnvalue from `parseArgs(argv, options);`, as I don't see any way to provide defaults to the parseArgs method.

If so, is it ok to use stuff like desctructuring in build.js, given that pretty much all the code in there now looks like ES5

Reply:
Yes, that’s fine with me; the runtime must be ES3 but the build script can use modern node.